backend/src/main/java/io/debezium/configserver/model/ConnectorType.java

```diff
@@ -5,26 +5,23 @@
  */
 package io.debezium.configserver.model;
 
-import java.util.List;
-
 public class ConnectorType {
 
     public String id;
     public String className;
     public String displayName;
     public String version;
     public boolean enabled;
-    public List<ConnectorProperty> properties;
 
     public ConnectorType() {
     }
 
-    public ConnectorType(String id, String className, String displayName, String version, boolean enabled, List<ConnectorProperty> properties) {
+    public ConnectorType(String id, String className, String displayName, String version, boolean enabled) {
```

Comment:
In `io.debezium.configserver.service.ConnectorIntegrator` the method that returns instances of this class is called `getDescriptor()`.  WDYT @mdrillin about renaming this class to `ConnectorDescriptor`?